### PR TITLE
Update building procedure

### PR DESCRIPTION
**Pull request by steven-gueguen**

### CODE REVIEW

File: BUILDING.md [hunk 1]
```diff
@@ -61,24 +20,18 @@ Add your API key & credentials in the gradle.properties file.
 
 ### Git tag
 
-commit & push the change.
+Commit & push the change.
 
-adds GIT tag `git tag -a 0.0.2-rc1 -m 'release candidate 0.0.2'`
+Adds GIT tag `git tag -a 0.0.2-rc1 -m 'release candidate 0.0.2'`
```
Comment (ftence) [inline BUILDING.md:25]: Add

File: BUILDING.md [hunk 2]
```diff
@@ -1,55 +1,14 @@
...
 
-## Install Thrift on MAC OS
+## Install Thrift
+You need Thrift in order to build the Warp 10 platform.
+Currently, Warp 10 use Thrift 0.11.0
 
-The current version of thrift used by Warp 10 is thrift 0.11.0
-  
-Then add "/opt/thrift-0.11.0/bin" to your path and add `export THRIFT_HOME=/opt/thrift-0.11.0`
+Install it and then add "/opt/thrift-0.11.0/bin" to your path and add `export THRIFT_HOME=/opt/thrift-0.11.0`
```
Comment (ftence) [inline BUILDING.md:7]: You just have to make sure `thrift` in in your path or you can define `THRIFT_HOME` if you don't want to. No need to do both. 

Also, the install may not be in `/opt`.

**Review by ftence — CHANGES_REQUESTED** [review]

File: BUILDING.md (hunk 1 above)
Comment (steven-gueguen) [reply BUILDING.md]: done

File: BUILDING.md (hunk 2 above)
Comment (steven-gueguen) [reply BUILDING.md]: done

**Review by ftence — APPROVED** [review]

**Pull request merged**